waiverdb/api_v1.py

```diff
@@ -134,6 +143,39 @@ def _filter_out_obsolete_waivers(query):
     return query.filter(Waiver.id.in_(subquery))
 
 
+def _verify_authorization(user, testcase):
+    if not permissions():
+        return True
+
+    ldap_host = current_app.config.get('LDAP_HOST')
+    ldap_searches = current_app.config.get('LDAP_SEARCHES')
+    if not ldap_searches:
+        ldap_base = current_app.config.get('LDAP_BASE')
+        if ldap_base:
+            ldap_search_string = current_app.config.get(
+                'LDAP_SEARCH_STRING', '(memberUid={user})'
+            )
+            ldap_searches = [{'BASE': ldap_base, 'SEARCH_STRING': ldap_search_string}]
+    return verify_authorization(user, testcase, permissions(), ldap_host, ldap_searches)
+
+
+def _authorization_warning(request):
+    testcase = request.args.get("testcase")
+    if testcase:
+        user, _headers = waiverdb.auth.get_user(request)
```

Comment:
_headers is not used, why not _

Reply:
`_headers` is clearer than `_` (the usual placeholder for unused variable).

Reply:
Is there flake8 warning for _headers? Previously I used _ to ignore unused variable, _header is the new format without warning?

Reply:
> Is there flake8 warning for _headers?

If there is, it is disabled by default. But pylint would complain (if we use it or you have it integrated in an editor) if the variable does not start with `_`:

```
waiverdb/api_v1.py:165:14: W0612: Unused variable 'headers' (unused-variable)
```

> Previously I used _ to ignore unused variable, _header is the new format without warning?

Often it is unclear what a function call would return, especially with tuples, so I prefer to use a named variable instead of confusing `_`.

Reply:
Ok. Thanks for explanation.